### PR TITLE
Add Data Explorer Params to the URL when user press run

### PR DESCRIPTION
**Pull request by rngus2344**

### CODE REVIEW

File: test/javascripts/acceptance/run-query-test.js.es6 [hunk 1]
```diff
@@ -191,4 +192,19 @@ acceptance("Data Explorer Plugin | Run Query", function (needs) {
       "the chart was rendered"
     );
   });
-});
+
+  function decodeQueryParam(p) {
```
Comment (markvanlan) [inline test/javascripts/acceptance/run-query-test.js.es6:196]: Looks like this function isn't being used. Can you remove it?

**Review by markvanlan — COMMENTED** [review]
Let's see what the feedback on Meta is and then I'll have someone else from the team review it

File: test/javascripts/acceptance/run-query-test.js.es6 (hunk 1 above)
Comment (rngus2344) [reply test/javascripts/acceptance/run-query-test.js.es6:196]: Okay!

**Comment by 4ong** [comment]
@markvanlan Is it possible to use this PR already or are there any existing plans to transform it for the production? Thanks!

**Comment by markvanlan** [comment]
@4ong I will do a bit more testing to make sure it's working as expected, and merge if it's looking good.

**Review by markvanlan — APPROVED** [review]

**Comment by 4ong** [comment]
Awesome, thanks - it's very timely for me!

**Pull request merged**